src/sumologic-span-processor/document-visibility-state.ts

```diff
@@ -83,7 +97,7 @@ export const onEnd = (readableSpan: ReadableSpan): void => {
         attributes: undefined,
         time: timestampInHrTime,
       });
-      if (state === 'hidden' && span.attributes[ATTRIBUTE_NAME] === 'visible') {
+      if (state === 'hidden') {
```

Comment:
Can you explain this logic and the change from the previous one?

Reply:
`state` can be either `visible` or `hidden`. I removed the second condition because it really does nothing here. Doing `span.attributes[ATTRIBUTE_NAME] = 'hidden';` multiple times won't hurt us, but simplifies our code.